app/models/admin/edition_filter.rb

```diff
@@ -117,6 +117,7 @@ def unpaginated_editions
       editions = editions.from_date(from_date) if from_date
       editions = editions.to_date(to_date) if to_date
       editions = editions.only_broken_links if only_broken_links
+      editions = editions.without_locked_documents
```

Comment:
Will this also exclude locked documents from the index page?  If this is the case, then it may be worth doing the part of the card to maintain this functionality as part of this PR, so functionality is not lost.

Reply:
Sure, was planning on raising a separate PR, but can add to this one 👍

Reply:
Now added :)